### ISSUE TITLE
App crashes when clicking on compose fab

### ISSUE THREAD

**Issue by aaronsbytes** [thunderbird/thunderbird-android#7842]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.900

### Where did you get the app from?

Other

### Android version

GrapheneOS / Android 14

### Device model

Google Pixel 6

### Steps to reproduce

1. Open the app
2. Click on the compose fab

### Expected behavior

The compose screen should appear.

https://github.com/thunderbird/thunderbird-android/assets/118199015/1fa72002-fde5-47de-9768-8cbcc9ce6d11

### Actual behavior

The app crashes immediately. I also tried to:

- Enable exploit protection compatability mode (GrapheneOS specific feature)
- Clear the storage & cache
- Reinstall the app

### Logs

```
type: crash
osVersion: google/oriole/oriole:14/AP1A.240505.004/2024050900:user/release-keys
package: com.fsck.k9:39000
process: com.fsck.k9
processUptime: 1017 + 187 ms
installer: com.android.packageinstaller

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.fsck.k9/com.fsck.k9.activity.MessageCompose}: java.lang.ClassCastException: com.google.android.material.button.MaterialButton cannot be cast to android.widget.ImageButton
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3957)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4137)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:108)
	at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:195)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:157)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:90)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2551)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8532)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ExecInit.main(ExecInit.java:50)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:359)
Caused by: java.lang.ClassCastException: com.google.android.material.button.MaterialButton cannot be cast to android.widget.ImageButton
	at com.fsck.k9.ui.compose.QuotedMessageMvpView.<init>(QuotedMessageMvpView.java:43)
	at com.fsck.k9.activity.MessageCompose.onCreate(MessageCompose.java:325)
	at android.app.Activity.performCreate(Activity.java:8820)
	at android.app.Activity.performCreate(Activity.java:8798)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1524)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3939)
	... 15 more
```

**Comment by anzepintar** [comment]
Reproduced/same issue in #7843, check that issue if you need in app debuging logs

**Comment by cketti** [comment]
Fixed in 6.901
